### PR TITLE
Customer managed deployments doc

### PR DESCRIPTION
**Pull request by arunkumar611**
Doc page for the new feature launch

Closed previous draft PR: https://github.com/pulumi/pulumi-hugo/pull/3798

### CODE REVIEW

File: themes/default/content/docs/pulumi-cloud/deployments/customer-managed-agents.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+title: Customer-Managed Agents
+title_tag: Get started with Customer-Managed Agents
+meta_desc: Customer-Managed Agents allows you to self-host deployment agents and get all the power and flexiblity of Pulumi Deployments in your isolated environments
+menu:
+  pulumicloud:
+    parent: deployments
+    weight: 2
+---
+
+Customer-managed deployments allow you to self-host deployment agents bringing the same power and flexibility as Pulumi-hosted deployments. Self-hosting your agents comes with many benefits:
```
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/deployments/customer-managed-agents.md:11]:
```suggestion
Customer-managed agents allow you to host your own Pulumi Cloud Deployments runners, bringing the same power and flexibility as Pulumi-hosted deployments. Self-hosting your agents comes with many benefits:
```

**Review by cnunciato — CHANGES_REQUESTED** [review]
It looks like this content is identical to the blog post, and having multiple looping videos isn't really suitable for a docs page, either. If there's any way we could provide some non-duplicate content for this page, that'd be ideal.

File: themes/default/content/docs/pulumi-cloud/deployments/customer-managed-agents.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Customer-Managed Agents support all the deployment triggers currently offered by Pulumi Deployments such as click to deploy, the Pulumi Deployments REST API, git push to deploy, Review Stacks, and remote Automation API.
+
+{{% notes "info" %}}
+Customer-Managed Agents is available on the Business Critical edition of Pulumi Cloud. [Contact sales](/contact/?form=sales) if you are interested and want to enable Customer-Managed Agents. If you are a self-hosted Pulumi Cloud customer, please [get in touch](https://share.hsforms.com/1YajiJ73sSuGn5RoTkyvKxg2mxud) to be notified when it is available.
+{{% /notes %}}
+
+## How to get started
+
+Before you begin, ensure you have installed the [Pulumi Github App](/docs/using-pulumi/continuous-delivery/github-app/) and update the [source control settings](/docs/pulumi-cloud/deployments/get-started). [Docker](https://docs.docker.com/engine/) is a requirement for the agent.
```
Comment (meagancojocar) [inline themes/default/content/docs/pulumi-cloud/deployments/customer-managed-agents.md:29]:
```suggestion
Before you begin, ensure you have installed the [Pulumi Github App](/docs/using-pulumi/continuous-delivery/github-app/) and update the [source control settings](/docs/pulumi-cloud/deployments/get-started). [Docker](https://docs.docker.com/engine/) is a requirement for using Pulumi Deployments.
```

**Review by meagancojocar — APPROVED** [review]
LGTM!

**Pull request closed without merging**